backend/controllers/auth.js

```diff
@@ -12,11 +13,20 @@ const register = async (req, res, next) => {
         username,
       },
     });
-    return res
-      .cookie("user", JSON.stringify(user), {
-        maxAge: COOKIE_MAX_AGE_MILISECONDS,
-      })
-      .json({ data: user });
+    const payload = {
+      id: user.id,
+    };
+    const token = jwt.sign(payload, JWT_SECRET, { expiresIn: "23h" });
```

Comment:
I think it's worth creating a separate token service module to reduce tight coupling between jwt package and authentication controllers. In case we need to make any changes to token generation logic, we'd need to do it in a single place, as opposed to current approach where jwt logic is used everywhere. 
Here is a suggestion:
Create a services folder with token.js file.
Create the necessary functions do operations on the jwt token. IT could look something like this

```js
function generateToken(payload) {
 return jwt.sign(/* ... */)
}
function verifyToken(token) {
 // ...
}

```